x/poolmanager/types/codec.go

```diff
@@ -13,13 +13,17 @@ import (
 func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&MsgSwapExactAmountIn{}, "osmosis/poolmanager/swap-exact-amount-in", nil)
 	cdc.RegisterConcrete(&MsgSwapExactAmountOut{}, "osmosis/poolmanager/swap-exact-amount-out", nil)
+	cdc.RegisterConcrete(&MsgSplitRouteSwapExactAmountIn{}, "osmosis/poolmanager/split-route-swap-exact-amount-in", nil)
+	cdc.RegisterConcrete(&MsgSplitRouteSwapExactAmountOut{}, "osmosis/poolmanager/split-route-swap-exact-amount-out", nil)
```

Comment:
cdc.RegisterConcrete has a bug where it would not be able to support sign on ledgers if it is over 40 characters (cref: https://github.com/cosmos/cosmos-sdk/issues/10870), we need to change to a shorter name.

On that note, I think it's time we backport this fix, I can try tackling it once CL audit is finished, but for now, lets use a shorted message name